projects/demo-integrations/src/tests/component-testing/predicate/predicate.cy.ts

```diff
@@ -0,0 +1,66 @@
+import {MaskitoOptions} from '@maskito/core';
+
+import {TestInput} from '../utils';
+
+describe('@maskito/angular | Predicate', () => {
+    const cardMask: MaskitoOptions = {
+        mask: [
+            ...new Array(4).fill(/\d/),
+            ' ',
+            ...new Array(4).fill(/\d/),
+            ' ',
+            ...new Array(4).fill(/\d/),
+            ' ',
+            ...new Array(4).fill(/\d/),
+        ],
+    };
+
+    const nameMask: MaskitoOptions = {
+        mask: /^[a-zA-Z\s]+$/,
+        postprocessors: [
+            ({value, selection}) => ({value: value.toUpperCase(), selection}),
+        ],
+    };
+
+    describe('can detect run-time changes', () => {
```

Comment:
Unfortunately, the new version of tests (inside this spec) are not equivalent to previous ones.

For these tests you should create the separate test component with the following template:
https://github.com/taiga-family/maskito/blob/5ac91a5d648bbce753cca0568b51ed66dbc81517/projects/demo/src/pages/cypress/examples/1-predicate/template.html#L1-L13

Reply:
done